src/Sylius/Bundle/AdminBundle/Twig/GridExtension.php

```diff
@@ -19,7 +19,7 @@
 
 final class GridExtension extends AbstractExtension
 {
-    public function __construct (
+    public function __construct(
         private readonly GridRendererInterface $gridRenderer,
     ) {
```

Comment:
```suggestion
    public function __construct(private readonly GridRendererInterface $gridRenderer) 
    {
```

Reply:
That kind of changes shouldn't be part of this PR, as we are unnecessarly extending the scope of it.